internal/vxproto/proto_test.go

```diff
@@ -1470,6 +1471,134 @@ func TestIMCSendRecvPackets(t *testing.T) {
 	wg.Wait()
 }
 
+func TestIMCSendRecvPacketsToTopic(t *testing.T) {
+	ctx := context.Background()
+	proto, err := getVXProto()
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer proto.Close(ctx)
+
+	topicName := "test_topic"
+	topicToken := proto.MakeIMCTopic(topicName, groupID)
+	module1Token := proto.MakeIMCToken("test1", groupID)
+	moduleSocket1 := proto.NewModule("test1", groupID)
+	if moduleSocket1 != nil {
+		if !proto.AddModule(moduleSocket1) {
```

Comment:
With testify library tests would become shorter and more readable, for example, for this 7 lines it can be rewritten to:
```
require.NotNil(t, moduleSocket1)
require.True(t, proto.AddModule(moduleSocket1), "failed to add sender Module Socket object")
```

Reply:
code should write in unified style and this will require rewriting all the tests